rustworkx/rustworkx.pyi

```diff
@@ -1423,6 +1423,7 @@ class PyDiGraph(Generic[_S, _T]):
     def in_edges(self, node: int, /) -> WeightedEdgeList[_T]: ...
     def incident_edge_index_map(self, node: int, /, all_edges: bool = ...) -> EdgeIndexMap: ...
     def incident_edges(self, node: int, /, all_edges: bool = ...) -> EdgeIndices: ...
+    def in_edge_indices(self, node: int, /) -> EdgeIndices: ...
```

Comment:
Please add this method to `PyGraph` as well. It is dumb BUT it doesn't hurt to keep the API consistent.

It will just return all edges for the `PyGraph` case and you should note that in the docstring